test/integration/options/posixExitCodes.spec.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var helpers = require('../helpers');
+var runMocha = helpers.runMocha;
+
+describe('--posix-exit-codes', function () {
+  // subprocess
+  var mocha;
+
+  function killSubprocess() {
+    mocha.kill('SIGKILL');
+  }
+
+  // these two handlers deal with a ctrl-c on command-line
+  before(function () {
+    process.on('SIGINT', killSubprocess);
+  });
+
+  after(function () {
+    process.removeListener('SIGINT', killSubprocess);
+  });
+
+  describe('when enabled with node options', function () {
+    it('should exit with code 134 on SIGABRT', function (done) {
```

Comment:
[Testing] To be thorough, could you add another test for a different signal? Just to make sure we're covering a couple, as the code should change between signals.

Also, there should be corresponding tests for what happens when `--posix-exit-codes` _isn't_ provided.